### DISCUSSION TITLE
Version compatible with IE11

### DISCUSSION THREAD

**Discussion by enboig** [gildas-lormeau/zip.js#280] [Q&A]
I am trying to use it in IE11 to upload compressed&passwored files to a server; but the most simple code I have working in modern browsers fail.
Which is the last version compatible with IE11? Is there a simple example (just an <input> which download the file in a zipped file)?

Thanks

**Comment by gildas-lormeau** [comment]
There is no version of zip.js supporting encrypted files officially compatible with IE11. I guess it should be possible to compile the code with babel to make it work with IE11 though.

**Comment by gildas-lormeau** [comment]
I'm moving this question into the "Discussions" tab since IE11 is not officially supported by zip.js.
